test/field_element_test.dart

```diff
@@ -0,0 +1,222 @@
+import 'package:solana/src/edwards25519/field_element.dart';
+import 'package:test/test.dart';
+
+void main() {
+  // Test vectors below, and the tests they are used in, are from
+  // curve25519-dalek.
+  // https://github.com/dalek-cryptography/curve25519-dalek/blob/4bdccd7b7c394d9f8ffc4b29d5acc23c972f3d7a/src/field.rs#L280-L301
+
+  // Random element a of GF(2^255-19), from Sage
```

Comment:
Since it's doc (versus a common comment) it should be `///`.

Reply:
Ok, will change it. It was in the original code as well so I didn't pay attention!